srcpkgs/OTPClient/patches/fix-cflags.patch

```diff
@@ -1,17 +1,15 @@
-Index: OTPClient-2.4.2/CMakeLists.txt
-===================================================================
---- OTPClient-2.4.2.orig/CMakeLists.txt
-+++ OTPClient-2.4.2/CMakeLists.txt
-@@ -11,11 +11,10 @@ option(BUILD_GUI "Build the GUI" ON)
+--- CMakeLists.txt	2021-04-27 06:27:11.708380752 +0300
++++ CMakeLists.txt	2021-04-27 06:28:14.020048767 +0300
```

Comment:
Would you mind keeping this and `patch_args` in the template? Makes the patches easier to work with.

Reply:
I reverted them back